app/src/composables/use-relation/use-relation-multiple.ts

```diff
@@ -270,6 +274,26 @@ export function useRelationMultiple(
 			else create(...selected);
 		}
 
+		function clear() {
+			if (!relation.value) return;
+
+			if (fetchedItems.value.length === 0) {
+				value.value = itemId.value === '+' ? undefined : [];
+				return;
+			}
+
+			const pkField =
+				relation.value.type === 'o2m'
+					? relation.value.relatedPrimaryKeyField.field
+					: relation.value.junctionPrimaryKeyField.field;
+
+			target.value.create = [];
+			target.value.update = [];
+			target.value.delete = fetchedItems.value.map((item) => item[pkField]);
```

Comment:
Should clear stage every item for deletion, or should this be `[]` as well with a call elsewhere to refresh the fetched items?